packages/contracts/contracts/L1/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >0.5.0 <0.8.0;
+pragma solidity >=0.8.4 <=0.9;
```

Comment:
You should be able to do `^0.8.4`, see https://docs.soliditylang.org/en/v0.8.6/layout-of-source-files.html#version-pragma